tools/regression.sh

```diff
@@ -764,7 +764,8 @@ for t in $tests; do
         -e 's|^/tmp/[0-9][0-9]*\.||'\
         -e 's/^Entity: line [0-9][0-9]*: //'\
         -e 's/schemas\.c:\([0-9][0-9]*\)/schemas.c:NNN/' \
-        -e 's/\(validation ([0-9][0-9]* of \)[0-9][0-9]*\().*\)/\1X\2/' $test_home/regression.$t.out
+        -e 's/\(validation ([0-9][0-9]* of \)[0-9][0-9]*\().*\)/\1X\2/' \
+        -e '/meta-data/d' $test_home/regression.$t.out
```

Comment:
To make this easier to add/remove patterns in the future, let's add another line break before the filename.